internal/cli/run.go

```diff
@@ -183,6 +183,11 @@ func runController() {
 
 	for _, cfg := range hostConfigs {
 		experiment.HostConfigs[cfg.Name] = cfg
+		if cfg.Location == "" {
+			cfg.Location = hotstuff.DefaultLocation
+		}
+		err := checkHostConfigLocations(cfg.Location)
+		checkf("invalid location configuration for host %s: %v", cfg.Name, err)
```

Comment:
Seem like this is a bit redundant: 
- the location is printed twice
- the error message doesn't add much value since the returned error already say `unknown location: %s`

Suggest to change this one to:
```
checkf("invalid configuration: %v", err)
```